packages/hoppscotch-common/src/pages/index.vue

```diff
@@ -279,9 +298,12 @@ function setupTabStateSync() {
       const tabStateFromSync =
         await platform.sync.tabState.loadTabStateFromSync()
 
-      if (tabStateFromSync) {
+      if (tabStateFromSync && !confirmSync.value.initialSync) {
         tabStateForSync.value = tabStateFromSync
-        confirmSync.value = true
+        changeCurrentSyncStatus({
+          initialSync: true,
+          sync: false,
+        })
       }
```

Comment:
From what i understand, Here the user is not actually choosing the initialSync, but i belive we're firing the `changeCurrentSyncStatus` to trigger the watcher that looks for `confirmSync.value.initialSync`, which shows the toast to the user. instead we can just move the content of that watcher to a function and call it.

If there's something i'm missing here, do let me know. 

```suggestion
      if (tabStateFromSync && !confirmSync.value.initialSync) {
        tabStateForSync.value = tabStateFromSync

        showSyncToast()
      }
```